aldryn_jobs/views.py

```diff
@@ -6,12 +6,13 @@
 from django.http import Http404
 from django.shortcuts import redirect
 from django.utils.translation import (
-    ugettext_lazy as _, get_language_from_request
+    ugettext as _, get_language_from_request
```

Comment:
I'm not a fan of overloading the ```_```.
My reasoning is that 90% of the cases ```_``` is used to represent a lazy translation string (```_lazy```) so it can be a bit confusing and even cause problems for someone assuming this. Instead if using a non lazy translation string, I always just import it explicitly instead of the as format.

So ```from django.utils.translation import ugettext```